### PR TITLE
Release process

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | all

This page is intended to replace the old  Release process and roadmap page: https://doc.ez.no/display/MAIN/eZ+Platform+Release+Process+and+Roadmap

When this is merged hopefully we can remove the original page and archive the whole Confluence space.

**Note**: The "Want to contribute" section was not copied here, as we have pages dedicated to contributing just below this one in the doc.

### CODE REVIEW

**Comment by DominikaK** [comment]
ping @rolandbenedetti , @andrerom 
FYI, the first commit is an almost direct copy of the text from Confluence, and in the second one I suggest some changes to the language, more in line with what we have in the rest of the doc, but please feel free to disagree :)

File: docs/community_resources/release_process.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# eZ Platform Release Process and Roadmap
+
+## Release process
+
+eZ Platform is distributed in two flavors:
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: eZ Platform comes in two distributions / has two distributions

File: docs/community_resources/release_process.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- eZ Platform Enterprise Edition (EE) is commercial software available under eZ Business User License (BUL) to eZ Enterprise subscribers. It is comprised of eZ Platform Open Source edition, additional enterprise support and maintenance services, as well as additional features which are not available in the open source software.
+
+We manage the release of eZ Platform using an agile iterative process and a continuous software development model, which is why we provide two kinds of eZ Platform releases:
+
+- Long Term Support releases (LTS) which are supported by eZ Systems for a long period of time (see [support lifecycle below](#support-lifecycle)), for eZ Enterprise subscribers.
+- Fast Track releases (FT) give access to the latest features and are supported for a short period of time, only until the next FT release is introduced. These are supported for both the open source community and eZ Enterprise subscribers.
+
+FT releases are tailored for those who want to stay more on the bleeding edge, while LTS releases are suitable for highly stable enterprise rollouts.
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: who want to stay `up-to-date with newest functionalities`, while

File: docs/community_resources/release_process.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+We usually release eZ Platform four times a year following the seasons (Winter, Spring, Summer and Fall). This usually includes one LTS release and three FT releases.
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: winter, spring, summer and fall

File: docs/community_resources/release_process.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Versioning conventions
+
+Both eZ Platform editions use [semantic versioning](http://semver.org/).  
+
+The version number of eZ Platform and all its internal components follows the semantic versioning conventions: vX.Y.Z.
+
+- Changes to X indicate a breaking change to the API, and developers know that they may need to implement a re-write of their code to continue using the API.
+- Y represents new features and functionality.
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: functionalities

File: docs/community_resources/release_process.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- Z represents patches, bug fixes, smaller improvements to unblock users, etc.
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: to unblock users - remove

File: docs/community_resources/release_process.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+This way, distribution files of our product are as follows:
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: Distribution files of our two editions are as follows:

File: docs/community_resources/release_process.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+- eZ Platform uses its own unique semantic version number: ezplatform-vX.Y.Z.tgz
+- eZ Platform Enterprise uses its semantic version number: ezplatformenterprise-vX.Y.Z.tgz
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: - for eZ Platform: ezplatform-vX.Y.Z.tgz
- for eZ Platform Enterprise: ezplatformenterprise-vX.Y.Z.tgz

File: docs/community_resources/release_process.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+## Support lifecycle
+
+Our software products are continuously evolving.
+With each release we strive to release stable products with cutting-edge technology.
+This means there is need for software maintenance services to provide bug fixes and adjustments for evolving web technologies.
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: for evolving web technologies - remove

File: docs/community_resources/release_process.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+As our products always provide new features and possibilities, our documentation and user forums may not always be able to provide an answer to all questions that may arise.
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: As our `products constantly provide` new features and possibilities

File: docs/community_resources/release_process.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
+# eZ Platform Release Process and Roadmap
```
Comment (MagdalenaZuba) [inline docs/community_resources/release_process.md]: eZ Platform release process and roadmap

File: docs/community_resources/release_process.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+- eZ Platform Enterprise Edition (EE) is commercial software available under eZ Business User License (BUL) to eZ Enterprise subscribers. It is comprised of eZ Platform Open Source edition, additional enterprise support and maintenance services, as well as additional features which are not available in the open source software.
+
+We manage the release of eZ Platform using an agile iterative process and a continuous software development model, which is why we provide two kinds of eZ Platform releases:
+
+- Long Term Support releases (LTS) which are supported by eZ Systems for a long period of time (see [support lifecycle below](#support-lifecycle)), for eZ Enterprise subscribers.
+- Fast Track releases (FT) give access to the latest features and are supported for a short period of time, only until the next FT release is introduced. These are supported for both the open source community and eZ Enterprise subscribers.
```
Comment (andrerom) [inline docs/community_resources/release_process.md]: `only until the next FT release is introduced` => `maintained only until the next FT release is introduced`

context: https://support.ez.no/Public/Service-Life  As in with exception of 2.2 they all have 3 months maintenance + 3 months support.

File: docs/community_resources/release_process.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+We usually release eZ Platform four times a year following the seasons (winter, spring, summer and fall). This usually includes one LTS release and three FT releases.
+
+## Versioning conventions
+
+Both eZ Platform editions use [semantic versioning](http://semver.org/).  
+
+The version number of eZ Platform and all its internal components follows the semantic versioning conventions: vX.Y.Z.
+
+- Changes to X indicate a breaking change to the API, and developers know that they may need to implement a re-write of their code to continue using the API.
```
Comment (andrerom) [inline docs/community_resources/release_process.md]: Maybe rather
`- Changes to X indicate a breaking changes, and while it is usually mostly internal things that change, developers know that they may need to see our change logs if they need to adjust their code to continue using the API / Features. If there are larger breaks, like v2 with admin ui, this is announced well in advance of the upcoming release.`

WDYT @rolandbenedetti Maybe to verbose here, but point is to not imply API changes a lot for every major.

**Pull request merged**